view-creator-framework/src/test/resources/sample-view-generation-spec.conf

```diff
@@ -52,6 +52,11 @@ pinotRealtime = {
       bufferTimePeriod = "12h"
     }
   }
+
```

Comment:
where do we configure the aggregation as SUM or is it implied at the moment ?

Reply:
It's implied. That's the only aggregation supported at ingestion time.